.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+# Describe your changes
+
+## Issue ticket number and link
+
+## Checklist before requesting a review
+
+- [ ] I have performed a self-review of my code.
+- [ ] If it is a core feature, I have added thorough tests.
+- [ ] - [ ] CHANGELOG.md has been updated should there be relevant changes in this PR.
```

Comment:
Copy and paste mistake here, an extra ` - [ ]`?

Reply:
Thanks for spotting it!